asreviewcontrib/statistics/statistics.py

```diff
@@ -123,6 +129,14 @@ def __str__(self):
         stat_str += "-----------    ATD    -----------\n"
         stat_str += f"{results['loss']: .3g}\n\n"
 
+        stat_str += f"Time to discovery:\n\n"
```

Comment:
would it be possible to add titles?

Reply:
No this is not possible because state files don't contain human-readable data (only features). (Different story for project files btw).